src/EnergyPlus/HybridModel.cc

```diff
@@ -0,0 +1,153 @@
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+
+// EnergyPlus Headers
+#include <HybridModel.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalance.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataRoomAirModel.hh>
+#include <General.hh>
+#include <HeatBalanceManager.hh>
+#include <InputProcessor.hh>
+#include <ScheduleManager.hh>
+#include <UtilityRoutines.hh>
+
+namespace EnergyPlus {
+
+namespace HybridModel {
```

Comment:
As the hybrid model is built upon inverse heat balance algorithms for each timestep and this relies on many variables, thus we think ZoneTempPredictorCorrector::CorrectZoneAirTemp is the place to add hybrid model algorithms to minimize development efforts. Still it is good to keep HybridModel module to take care new object inputs, process HybridStartDayOfYear and HybridEndDayOfYear, and create hybrid model flags.